seqr/utils/elasticsearch/es_utils_tests.py

```diff
@@ -1990,6 +1990,35 @@ def test_multi_dataset_get_es_variants(self):
             dict(filters=[path_filter, ALL_INHERITANCE_QUERY], start_index=0, size=5, index=INDEX_NAME),
         ])
 
+    @urllib3_responses.activate
+    def test_multi_dataset_no_affected_inheritance_get_es_variants(self):
+        setup_responses()
+        # The family has multiple data types loaded but only one loaded in an affected individual
+        Sample.objects.get(individual_id=4, elasticsearch_index=SV_INDEX_NAME).delete()
+
+        search_model = VariantSearch.objects.create(search={'inheritance': {'mode': 'de_novo'}})
+        results_model = VariantSearchResults.objects.create(variant_search=search_model)
+        results_model.families.set(Family.objects.filter(guid='F000002_2'))
+
+        get_es_variants(results_model, num_results=2)
+        self.assertExecutedSearch(filters=[{'bool': {
```

Comment:
Is any test for the `index_name` set correctly?

Reply:
yes, `assertExecutedSearch` checks the index name and unless you specify otherwise it checks for the default `test_index`, which is what we want here